src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -33,8 +35,36 @@ const styles = StyleSheet.create({
 });
 ```
 
+</Block>
+<Block name="React Native CLI">
+
+```js
```

Comment:
React code should probably use jsx syntax highlighting. Is it supported?

Reply:
yes it is supported. I have updated it to use jsx for React syntax highlighting.